### PR TITLE
Allow spaces in name attributes.

### PR DESCRIPTION
**Pull request by gfl-chris**

### CODE REVIEW

File: opsgenie/integration_util.go [hunk 1]
```diff
@@ -33,9 +33,9 @@ func expandOpsgenieIntegrationResponders(d *schema.ResourceData) []integration.R
 
 func validateOpsgenieIntegrationName(v interface{}, k string) (ws []string, errors []error) {
 	value := v.(string)
-	if !regexp.MustCompile(`^[a-zA-Z0-9_-]+$`).MatchString(value) {
```
Comment (ffahri) [inline opsgenie/integration_util.go:36]: this line is not correct statement in terms of regex

File: opsgenie/resource_opsgenie_schedule.go [hunk 2]
```diff
@@ -164,9 +164,9 @@ func resourceOpsgenieScheduleDelete(d *schema.ResourceData, meta interface{}) er
 
 func validateOpsgenieScheduleName(v interface{}, k string) (ws []string, errors []error) {
 	value := v.(string)
-	if !regexp.MustCompile(`^[a-zA-Z0-9_-]+$`).MatchString(value) {
+	if !regexp.MustCompile(`^[a-zA-Z0-9_- ]+$`).MatchString(value) {
```
Comment (ffahri) [inline opsgenie/resource_opsgenie_schedule.go:167]: Same as integrationutil also not correct

File: opsgenie/resource_opsgenie_team.go [hunk 3]
```diff
@@ -197,9 +197,9 @@ func expandOpsGenieTeamMembers(d *schema.ResourceData) []team.Member {
 
 func validateOpsGenieTeamName(v interface{}, k string) (ws []string, errors []error) {
 	value := v.(string)
-	if !regexp.MustCompile(`^[a-zA-Z0-9_-]+$`).MatchString(value) {
+	if !regexp.MustCompile(`^[a-zA-Z0-9_- ]+$`).MatchString(value) {
```
Comment (ffahri) [inline opsgenie/resource_opsgenie_team.go:200]: Also same as other two

**Review by ffahri — CHANGES_REQUESTED** [review]

**Comment by ffahri** [comment]
Thank you for pointing about this bug

**Comment by ffahri** [comment]
It should be good if you change it to `a-zA-Z 0-9_-` (space in the middle)

**Comment by ffahri** [comment]
the fix has added to master thank you for help

**Comment by gfl-chris** [comment]
Thank you. :+1:

**Pull request closed without merging**